example-code/cypress/pages/confirm.js

```diff
@@ -0,0 +1,65 @@
+export const confirm ={
+    TXT_NAME: "table >tbody >tr:nth-child(1) td:nth-child(2)",

```

Comment:
You can try `TXT_VALUE: "td + td"` then` cy.get(this.TXT_VALUE).contains(value).should("be.visible")`. You can use just one locator to implement all verify because you know how the result should be